ceph.spec.in

```diff
@@ -82,10 +84,10 @@ ExclusiveArch:  x86_64 aarch64
 #################################################################################
 # dependencies that apply across all distro families
 #################################################################################
-Requires:       ceph-osd = %{epoch}:%{version}-%{release}
-Requires:       ceph-mds = %{epoch}:%{version}-%{release}
-Requires:       ceph-mgr = %{epoch}:%{version}-%{release}
-Requires:       ceph-mon = %{epoch}:%{version}-%{release}
+Requires:       ceph-osd = %{?epoch:%{epoch}:}%{version}-%{release}
```

Comment:
I think we can, and `_epoch` is a good name for it. I just don't know whether to use `%define` or `%global` ?